pom.xml

```diff
@@ -1470,5 +1470,31 @@
       </build>
     </profile>
 
+    <profile>
+      <id>pitest-size</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.pitest</groupId>
+            <artifactId>pitest-maven</artifactId>
+            <version>1.1.10</version>
```

Comment:
Since we'' have multiple profiles for each run, lets make this version number a property so we only have to define it once.